CHANGELOG.md

```diff
@@ -3,6 +3,18 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and
 the [“Keep a Changelog” format](http://keepachangelog.com/).
 
+
+## Upcoming
+### Added
+This version introduces a new functionality:
+
+- `extension.dialogs.select...(options)` opens a dialog for selecting entries
+  and assets
+
+You can now open the native entity selection dialog window from your extension.
+`options` object passed to the call allows to configure behavior of search and
+insertion. Please see API reference for more details.
```

Comment:
@jelz Which API? This one?

https://www.contentful.com/developers/docs/references/content-management-api/#/reference/editor-interface